src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import './App.scss';
-
+import { TodoList } from './components/TodoList';
 import todos from './api/todos';
 import users from './api/users';
 
-function App() {
+const preparedTodos = todos.map(item => ({
+  ...item,
+  user: users.find(user => user.id === item.userId),
+}));
```

Comment:
todos - array
todo - one element
```suggestion
const preparedTodos = todos.map(todo => ({
  ...item,
  user: users.find(user => user.id === todo.userId),
}));
```